python3/amplify/tests/test.py

```diff
@@ -0,0 +1,549 @@
+        "sc-bytes": "335",
+        "c-ip": "80.179.110.98",
+        "cs-method": "GET",
+        "cs\\(Host)": "zzz.cloudfront.net",
+        "cs-uri-stem": "/manifest.json",
+        "sc-status": "304",
+        "cs\\(Referer)": "https://master.zzz.amplifyapp.com/",
+        "cs\\(User-Agent)": "Mozilla/5.0%20\\(Macintosh;%20Intel%20Mac%20OS%20X%2010_15_7)%20AppleWebKit/537.36%20\\(KHTML%20like%20Gecko)%20Chrome/102.0.0.0%20Safari/537.36",
+        "cs-uri-query": "-",
+        "cs\\(Cookie)": "-",
+        "x-edge-result-type": "RefreshHit",
+        "x-edge-request-id": "9YizjAMFz57jXWVOXhEpU20DdWrOcJdMZtxZObp1PLhiAoIHqYr8iw\\=\\=",
+        "x-host-header": "master.zzz.amplifyapp.com",
+        "cs-protocol": "https",
+        "cs-bytes": "444",
+        "time-taken": "0.273",
+        "x-forwarded-for": "-",
+        "ssl-protocol": "TLSv1.3",
+        "ssl-cipher": "TLS_AES_128_GCM_SHA256",
+        "x-edge-response-result-type": "RefreshHit",
+        "cs-protocol-version": "HTTP/2.0",
+        "fle-status": "-",
+        "fle-encrypted-fields": "-",
+        "c-port": "51112",
+        "time-to-first-byte": "0.273",
+        "x-edge-detailed-result-type": "RefreshHit",
+        "sc-content-type": "-",
+        "sc-content-len": "-",
+        "sc-range-start": "-",
+        "sc-range-end": "-",
+                "function_version": 1,
+                "invoked_function_arn": 1,
+        "type": "logzio_amplify_lambda"
+
+    },
+]
+
+
+fileConfig('amplify/tests/logging_config.ini')
+logger = logging.getLogger(__name__)
+
+
+class Context(object):
+    function_version = 1
+    invoked_function_arn = 1
+    memory_limit_in_mb = 128
+
+
+class TestLambdaFunction(unittest.TestCase):
+    """ Unit testing logzio lambda function """
+
+    def setUp(self):
+        # Set os.environ for tests
+        os.environ['TOKEN'] = "123456789"
+        os.environ['TYPE'] = "vpcflow"
+        self._logzioUrl = "https://listener.logz.io:8071/?token={}".format(
+            os.environ['TOKEN'])
+
+    def test_add_additional_logs(self):
+        try:
+            for i in range(len(DATA_LOG_SAMPLE)):
+                modified_log = worker._get_additional_logs_data(
+                    DATA_LOG_SAMPLE[i], Context)
+                self.assertEqual(
+                    modified_log['function_version'], DATA_LOG_SAMPLE_UPDATED_TIMESTAMP[i]['function_version'])
+                self.assertEqual(
+                    modified_log['invoked_function_arn'], DATA_LOG_SAMPLE_UPDATED_TIMESTAMP[i]['invoked_function_arn'])
+                if os.environ.get('TYPE'):
+                    self.assertEqual(
+                        modified_log['type'], os.environ.get('TYPE'))
+                else:
+                    self.assertEqual(
+                        modified_log['type'], 'logzio_amplify_lambda')
+        except Exception as e:
+            self.fail(
+                "Additional data is wrong")
+
+    def test_receive_data_from_request_to_array(self):
```

Comment:
Please make sure that the test's name indicates which function it tests.
For example here it tests `convert_csv_to_array_of_logs` so it should be `test_convert_csv_to_array_of_logs`.